src/cmd/sql_cmd_test.cc

```diff
@@ -1514,22 +1522,26 @@ TEST_P(DBSDKTest, DeployLongWindowsExecuteCountWhere) {
     std::string msg;
     CreateDBTableForLongWindow(base_db, base_table);
 
-    std::string deploy_sql = "deploy test_aggr options(long_windows='w1:2') select col1, col2,"
-        " count_where(i64_col, filter<1) over w1 as w1_count_where_i64_col_filter,"
-        " count_where(i64_col, col1='str1') over w1 as w1_count_where_i64_col_col1,"
-        " count_where(i16_col, filter>1) over w1 as w1_count_where_i16_col,"
-        " count_where(i32_col, 1<filter) over w1 as w1_count_where_i32_col,"
-        " count_where(f_col, 0=filter) over w1 as w1_count_where_f_col,"
-        " count_where(d_col, 1=filter) over w1 as w1_count_where_d_col,"
-        " count_where(t_col, 1>=filter) over w1 as w1_count_where_t_col,"
-        " count_where(s_col, 2<filter) over w1 as w1_count_where_s_col,"
-        " count_where(date_col, 2>filter) over w1 as w1_count_where_date_col,"
-        " count_where(col3, 0>=filter) over w2 as w2_count_where_col3"
-        " from " + base_table +
-        " WINDOW w1 AS (PARTITION BY col1,col2 ORDER BY col3"
-        " ROWS_RANGE BETWEEN 5 PRECEDING AND CURRENT ROW), "
-        " w2 AS (PARTITION BY col1,col2 ORDER BY i64_col"
-        " ROWS BETWEEN 6 PRECEDING AND CURRENT ROW);";
+    std::string deploy_sql =
+        R"(DEPLOY test_aggr options(long_windows='w1:2')
+    SELECT
+        col1, col2,
+        count_where(i64_col, filter<1) over w1 as w1_count_where_i64_col_filter,
+        count_where(i64_col, col1='str1') over w1 as w1_count_where_i64_col_col1,
+        count_where(i16_col, filter>1) over w1 as w1_count_where_i16_col,
+        count_where(i32_col, 1<filter) over w1 as w1_count_where_i32_col,
+        count_where(f_col, 0=filter) over w1 as w1_count_where_f_col,
+        count_where(d_col, 1=filter) over w1 as w1_count_where_d_col,
+        count_where(t_col, 1>=filter) over w1 as w1_count_where_t_col,
+        count_where(s_col, 2<filter) over w1 as w1_count_where_s_col,
+        count_where(date_col, 2>filter) over w1 as w1_count_where_date_col,
+        count_where(col3, 0>=filter) over w2 as w2_count_where_col3 from )" +
```

Comment:
possible to add ut for other filter type except the `int`?

Reply:
added in next test case